admin/views/tabs/metas/paper-content/front-page-content.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin\Views\PaperContent
+ *
+ * @uses Yoast_Form                               $yform                        Form object.
+ * @uses WP_Taxonomy                              $wpseo_post_type
+ * @uses Yoast_View_Utils                         $view_utils
+ * @uses WPSEO_Admin_Recommended_Replace_Vars     $recommended_replace_vars
+ */
+
+use Yoast\WP\SEO\Presenters\Admin\Badge_Presenter;
+
+$frontpage_settings_badge = new Badge_Presenter( 'frontpage_settings' );
+
+// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- Is correctly escaped in the present() method.
+echo '<div>' . $frontpage_settings_badge->present() . '</div>';
+
+echo '<p>' . esc_html__( 'These are the title, description and image used in the Open Graph meta tags on the front page of your site.', 'wordpress-seo' ) . '</p>';
+
+echo '<div id="yoast-og-frontpage-image-select"></div>';
+
+$yform->hidden( 'og_frontpage_image', 'og_frontpage_image' );
+$yform->hidden( 'og_frontpage_image_id', 'og_frontpage_image_id' );
+
+$editor = new WPSEO_Replacevar_Editor(
+	$yform,
+	[
+		'title'                   => 'og_frontpage_title',
+		'description'             => 'og_frontpage_desc',
+		'page_type_recommended'   => 'homepage',
+		'page_type_specific'      => 'page',
+		'paper_style'             => false,
+		'label_title'             => \esc_html__( 'Social title', 'wordpress-seo' ),
+		'label_description'       => \esc_html__( 'Social description', 'wordpress-seo' ),
+		'description_placeholder' => \esc_html__( 'Modify your social description by editing it right here.', 'wordpress-seo' ),
+	]
+);
+$editor->render();
+
+// Offer copying of meta description.
+$homepage_meta_description = WPSEO_Options::get( 'metadesc-home-wpseo' );
+
+if ( ! empty( $homepage_meta_description ) ) {
+	$copy_home_description_button_label = \esc_html__( 'Use meta description of your homepage', 'wordpress-seo' );
+
+	$copy_home_description_help = \sprintf(
+		/* translators: 1: link open tag; 2: link close tag., 3: the translated label of the button */
+		\esc_html__( 'Click the "%3$s" button below to use the meta description set in the %1$sGeneral%2$s tab which is used for your homepage.', 'wordpress-seo' ),
```

Comment:
In the design, this text is different and it's "Click the button below to use ...". It appears this was added at some point to I'm not sure whether any decision has been made that isn't in the design sketches? Cc @jcomack